service_mesh/service_mesh_install/preparing-ossm-installation.adoc

```diff
@@ -0,0 +1,21 @@
+[id="preparing-ossm-installation"]
+= Preparing to install {ProductName}
+include::modules/ossm-document-attributes.adoc[]
+:context: preparing-ossm-installation
+toc::[]
+
+Before you can install {ProductName}, review the installation activities, ensure that you meet the prerequisites, update the node configuration for {product-title} 3.11 instances, and update the container registry if you are running {product-title} 3.11 on-premise:
```

Comment:
Are you publishing this content in both 3.11 and 4.1? I think it's confusing to put references to 3.11 in the 4.1 collection.

Reply:
This is currently being discussed.

Reply:
We are pulling all the 3.11 content and I'll do an update to the master-3 branch sometime after TP12 goes out.